rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/azure/AzureBakeHandler.groovy

```diff
@@ -61,9 +62,11 @@ public class AzureBakeHandler extends CloudProviderBakeHandler{
     // TODO(duftler): Presently scraping the logs for the image name. Would be better to not be reliant on the log
     // format not changing. Resolve this by storing bake details in redis.
     logsContent.eachLine { String line ->
-      if (line =~ IMAGE_NAME_TOKEN) {
-        imageName = line.split("/").last()
-        ami = "https:" + line.split(":").last()
+      if (line.startsWith(IMAGE_NAME_TOKEN)) {
+        imageName = line.substring(IMAGE_NAME_TOKEN.size())
+      }
+      if (line.startsWith(IMAGE_ID_TOKEN)) {
+        ami = line.substring(IMAGE_ID_TOKEN.size())
```

Comment:
If there are two business scenarios mapped with these 2 cases, suggest renaming the two variables, at least renaming the "ami" since this is the AWS naming convention.

Reply:
Would be nice to rename it to something not aws-specific, but kind of stuck with it for historical reasons; the upstream services rely on that name for this object: https://github.com/spinnaker/orca/blob/master/orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/api/Bake.groovy#L36

On the GCE side, we don't use it, but that's only because there isn't a meaningful distinction between id and name for a vm image (well, it's not so much that there isn't a distinction, it's that the id isn't relied on and we address the images solely by name).

Reply:
In order not to break anything, I didn't change the naming of variable